pyomo/solvers/plugins/solvers/GLPK.py

```diff
@@ -19,6 +19,7 @@
 
 from pyomo.common import Executable
 from pyomo.common.collections import Bunch
+from pyomo.common.errors import ApplicationError
```

Comment:
FYI - I added this because, when I opened the file on my IDE, it immediately warned me that `ApplicationError` was used in the file but hadn't been imported.